src/app/components/cards/Comment.jsx

```diff
@@ -287,10 +287,11 @@ class CommentImpl extends React.Component {
         let controls = null;
 
         if (!this.state.collapsed && !hide_body) {
+            var sanitizedBody = comment.body.replace(/(\<\?{0,}php)/, 'php');
```

Comment:
Wouldn't this simply change all tags, including those in \`code` formatting for example?